src/fourier_rotate.jl

```diff
@@ -53,7 +55,8 @@ function rotate(arr, θ, rotation_plane=(1, 2); adapt_size=true, keep_new_size=f
                 0
             end
         end
-        arr = select_region(arr, new_size=old_size .+ extra_size, pad_value=pad_value)
+
+        arr = select_region_view(arr, new_size=old_size .+ extra_size, pad_value=pad_value)
```

Comment:
why is this changed to the `view`?

Reply:
... to avoid copying too much and to reduce memory requirements.

Reply:
did you benchmark? Did it change anything?
I would not change any unaffected code by this introduction.